src/pages/Address/AddressAssetComp.tsx

```diff
@@ -156,7 +187,15 @@ export const AddressNRC721Comp = ({ account }: { account: NRC721 }) => {
   )
 }
 
-export const AddressCoTAComp = ({ account }: { account: CoTA }) => {
+export const AddressCoTAComp = ({
+  account,
+  isRGBPP,
+  udtLabel,
```

Comment:
It's `cota comp` so the label should be `cota`

Reply:
done